### ISSUE TITLE
Accessing futures websocket data stream

### ISSUE THREAD

**Issue by vectar31** [dmikushin/binance-cxx-api#21]
Currently we have access to spot data which is available at 'stream.binance.com'. Can we access futures data also which is available at fstream.binance.com? Simply changing BINANCE_WS_HOST to fstream.binance.com doesn't work. Has anyone else tried accessing futures data using binance-cxx-api?

**Comment by iamwrm** [comment]
Yeah I have tried changing the BINANCE_WS_HOST  and also the endpoint of binance::Account::startUserDataStream to https://fapi.binance.com/fapi/v1/listenKey. It still doesn't work. I could successfully get a listenKey but the ws returns nothing.

**Comment by iamwrm** [comment]
@vectar31 Dude, change the `#define BINANCE_WS_PORT` from 9433 to 433 in `binance_websocket.h`  You are welcome.

**Comment by vectar31** [comment]
@iamwrm Thanks. I was able to connect to futures stream at port 443 not 433. Interestingly spot stream at stream.binance.com also works with port 443. I don't know why that happens. 

Also, have you tried making multiple socket connections with this? For example, one to stream.binance.com and the other to fstream.binance.com?
